courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>Same as above, I'd remove the "argument_name=" for all the positional arguments to avoid code verbosity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>